### PR TITLE
Add python packages: pyensembl and its dependencies, gffutils, pyBigWig

### PR DESCRIPTION
**Pull request by moritzschaefer**
##### Motivation for this change

Both pyenseml and gffutils are commonly used python packages (> 100 stars on GitHub), so I added them to nix packages. 

###### Things done

- Built on platform(s)
   - [x] NixOS
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`) (only pyensembl has a supplemental binary)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [-] Mostly Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md). I designed the derivations 1:1 to existing ones.

### CODE REVIEW

**Comment by FRidh** [comment]
Please rebase, don't merge master.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7343,6 +7343,27 @@ in {
...
+
+  pyensembl = callPackage ../development/python-modules/pyensembl { };
+
+  gffutils = callPackage ../development/python-modules/gffutils { };
+
+  scikit-plot = callPackage ../development/python-modules/scikit-plot { };
+
+  pybedtools = callPackage ../development/python-modules/pybedtools { };
+
+  pyBigWig = callPackage ../development/python-modules/pyBigWig { };
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix:7366]: normalized names (lowercase and dashes instead of underscores)

**Review by FRidh — CHANGES_REQUESTED** [review]
Please ensure the modules are somehow tested. Maybe supporting the unit tests is too much, but we have the `pythonImportsCheckHook` so you could do import checks at least.

Also, remove empty lists and strings.

Finally, will you maintain the new packages?

**Comment by moritzschaefer** [comment]
@FRidh thanks for pointing this out. The module 'pybigwig' already existed in the correct format (lowercase) so I just reverted that commit. 

I cannot guarantee to maintain the packages in a timely manner, but will try to

**Pull request closed without merging**